src/extensions/rate_limit/global.rs

```diff
@@ -159,6 +159,6 @@ mod tests {
         let duration = start.elapsed().as_millis();
         println!("duration: {} ms", duration);
         // should take between 800..900 millis. each 100ms period handles 10 calls
```

Comment:
```suggestion
        // should take between 800..1000 millis. each 100ms period handles 10 calls
```

Reply:
No this comment is right, but in debug, sometime will fail.